### DISCUSSION TITLE
How to use a custom header for example requests in JSON file?

### DISCUSSION THREAD

**Discussion by GregLeung** [specmatic/specmatic#918] [Ask For Help] [answered]
Hi, I am following the documentation for using EXTERNALIZED TEST DATA for java project, could you pls also provide an example which uses a custom header? cheers

**Comment by GregLeung** [comment, marked as answer]
I defined:

      "headers":{
          "header": "header Value"
      },

and it works

**Reply by harikrishnan83** [reply]
@GregLeung Brilliant stuff, thanks. Sorry we could not respond earlier.
